lib/node_runner_message_checker_test.go

```diff
@@ -0,0 +1,117 @@
+package sebak
+
+import (
+	"testing"
+
+	"github.com/stellar/go/keypair"
+	"github.com/stretchr/testify/assert"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/error"
+	"boscoin.io/sebak/lib/network"
+	"boscoin.io/sebak/lib/node"
+	"boscoin.io/sebak/lib/storage"
+)
+
+func MakeNodeRunner() (*NodeRunner, *sebaknode.LocalNode) {
+	kp, _ = keypair.Random()
+
+	nodeEndpoint := &sebakcommon.Endpoint{Scheme: "https", Host: "https://locahost:5000"}
+	localNode, _ := sebaknode.NewLocalNode(kp, nodeEndpoint, "")
+
+	vth, _ := NewDefaultVotingThresholdPolicy(66, 66)
+	is, _ := NewISAAC(networkID, localNode, vth)
+	st, _ := sebakstorage.NewTestMemoryLevelDBBackend()
+	network, _ := createNetMemoryNetwork()
+	nodeRunner, _ := NewNodeRunner(string(networkID), localNode, vth, network, is, st)
+	return nodeRunner, localNode
+}
+
+func TestMessageChecker(t *testing.T) {
+	var CheckerFuncs = []sebakcommon.CheckerFunc{
```

Comment:
If you wanted to test a separated checker func, we don't need to set the `CheckerFuncs` to make `Checker`.

Reply:
The `CheckerFuncs` used in https://github.com/spikeekips/sebak/compare/0new-proposer-bug-fix...Charleslee522:message-checker-unittest#diff-63d21af35564a3bffc703875fda355d5R85 :)

Reply:
@Charleslee522 
```
		DefaultChecker: sebakcommon.DefaultChecker{},
```
I guess this will be enough, I did not test :)

Reply:
```
 	DefaultChecker: sebakcommon.DefaultChecker{},
```

@spikeekips With this, the test is not passed.
And it is also changed that not include `CheckNodeRunnerHandleMessageTransactionBroadcast` in `CheckerFuncs`